lib/datasource/galaxy/index.ts

```diff
@@ -77,7 +77,7 @@ export class GalaxyDatasource extends Datasource {
     };
 
     result.dependencyUrl = galaxyProjectUrl;
-    const { github_user: user = null, github_repo: repo = null } = resultObject;
+    const { github_user: user, github_repo: repo } = resultObject;
```

Comment:
default value isn't required, we can use the undefined value here. Will be validated in next condition.